src/app/views/sidebar/sample-queries/SampleQueries.tsx

```diff
@@ -59,9 +61,11 @@ export class SampleQueries extends Component<ISampleQueriesProps, any> {
   }
 
   public componentDidMount = () => {
-    const { queries } = this.props.samples;
-    if (queries && queries.length > 0) {
-      this.setState({ sampleQueries: queries });
+    const { samples, permissionModeType } = this.props;
+    const queries = samples.queries;
+
+    if ((queries.sampleQueries && queries.sampleQueries.length > 0) || (queries.teamsAppSampleQueries && queries.teamsAppSampleQueries.length > 0)) {
```

Comment:
where are teamsAppSampleQueries coming from?

Reply:
Its the new list thats being utilized for the application mode sample queries